test/e2e/storage/volume_provisioning.go

```diff
@@ -408,6 +416,29 @@ var _ = SIGDescribe("Dynamic Provisioning", func() {
 			}
 		})
 
+		It("should provision storage with mount options", func() {
+			framework.SkipUnlessProviderIs("gce", "gke")
+
+			test := storageClassTest{
+				"HDD PD on GCE/GKE",
+				[]string{"gce", "gke"},
+				"kubernetes.io/gce-pd",
+				map[string]string{
+					"type": "pd-standard",
+				},
+				"1Gi",
+				"1Gi",
+				func(volume *v1.PersistentVolume) error {
+					return checkGCEPD(volume, "pd-standard")
+				},
+			}
+			class := newStorageClass(test, ns, "mountoptions")
+			class.MountOptions = []string{"debug"}
```

Comment:
is it worth specifying several mount options here?

Reply:
sure, added resgid=0, should be harmless.